crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -402,12 +400,19 @@ $env.config.ls = true
 # WARNING: A malformed display_output hook can suppress all Nushell output to the terminal.
 #          It can be reset by assigning an empty string as below:
 
-$env.config.hooks.pre_prompt = []          # Before each prompt is displayed
-$env.config.hooks.pre_execution = []       # After <enter> is pressed; before the commandline
-                                           # is executed
-$env.config.hooks.env_change = []          # When a specified environment variable changes
-$env.config.hooks.display_output = ""      # Before Nushell output is displayed in the terminal
-$env.config.hooks.command_not_found = []   # When a command is not found
+# Before each prompt is displayed
+$env.config.hooks.pre_prompt = [{ null }]
```

Comment:
I can understand moving the comments for consistency, but why the `null` additions?

Reply:
Those were in 0.100.0 example config, and because `$env.config.hooks.env_change` had wrong type, I decided to return default value as an example. I can remove all nulls if you prefer so